### DISCUSSION TITLE
New ChromeOS dont work

### DISCUSSION THREAD

**Discussion by SkylarAyr** [3kh0/ext-remover#77] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Version 106.0.5249.112 (Official Build) (64-bit) doesnt work this is new update, please make a new one I need this.

**Comment by 3kh0** [comment, marked as answer]
It is coming soon.

**Reply by Endernates** [reply]
Really? I could get it on ChromeOS.

**Reply by H454N-H** [reply]
I could

**Reply by MH4554NP** [reply]
sme

**Reply by raventhecrabraver** [reply]
hopefully soon enough

**Reply by MH4554NP** [reply]
Will we get it before christmas by any chance?

**Reply by 3kh0** [reply]
https://github.com/3kh0/ext-remover#for-v106-and-up it is here!

**Comment by H454N-H** [comment]
how long though

**Reply by Fallenmoon8080** [reply]
Be patient

**Reply by H454N-H** [reply]
ok

**Reply by HU5541N** [reply]
“Patience is bitter, but its fruit is sweet.” —Jean-Jacques Rousseau

**Reply by 3kh0** [reply]
https://github.com/3kh0/ext-remover#for-v106-and-up it is here!

**Comment by SomePVZPlayerMatt** [comment]
be patient

**Reply by H454N-H** [reply]
ok

**Reply by HU5541N** [reply]
cool down guys

**Reply by MH4554N** [reply]
"I know right!"     like Oh my god! so rude

**Reply by H454N-H** [reply]
Calm the beef Lads

**Reply by 3kh0** [reply]
https://github.com/3kh0/ext-remover#for-v106-and-up

**Reply by Aka-but-nice** [reply]
YAYAysyqsyqs im so happy jk this  shit don't work!

**Comment by H454N-H** [comment]
If anybody knows what senso.client is then you must feel my pain...😢

**Reply by HU5541N** [reply]
Sir i am going to have to ask you to calm down and wait patiently or there will be consequences

**Reply by H454N-H** [reply]
Who are you to tell me to wait you little boy. I will have You Know I am 34.  And I Work For the FBI so I WILL track you down to your house and I WILL find you and I WILL BOMB YOU!

**Reply by HU5541N** [reply]
Was That a Threat because i hope it wasnt because i dont want to be taking legal actions against you

**Reply by rwg100** [reply]
Hey, my dad works for the FBI and can get your entire neighborhood addicted to crack!

**Reply by Aka-but-nice** [reply]
I wonder how Pingot is gonna work, or the new ext remover

**Comment by MAmaan28** [comment]
can we please have an update on the new ext remover

**Reply by H454N-H** [reply]
Be patient you little immigrated little brat

**Reply by 3kh0** [reply]
Agree ong

**Reply by H454N-H** [reply]
what does that mean

**Reply by MH4554NP** [reply]
ong means on god

**Comment by Win-Rar2** [comment]
use unblockers like google.winrarisyou.repl.co

**Reply by rwg100** [reply]
username: win-rar 2
the link: **WIN RAR IS YOU**
